### PR TITLE
fix: skip partnerinformation on separated partner in finanzielleAngaben

### PR DESCRIPTION
**Pull request by ekl176**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208269580933971

### CODE REVIEW

File: app/flows/beratungshilfeFormular/finanzielleAngaben/flow.json [hunk 1]
```diff
@@ -116,7 +116,7 @@
                 "target": "#partner.unterhalt"
               },
               {
-                "guard": "hasPartnerschaftOrSeparated",
```
Comment (chohner) [inline app/flows/beratungshilfeFormular/finanzielleAngaben/flow.json:119]: looks like we could remove (or at least remove the export) of `hasPartnerschaftOrSeparated` :)

**Review by chohner — APPROVED** [review]
nice, tried it ✅ we could now even do some additional cleanup I think

File: app/flows/beratungshilfeFormular/finanzielleAngaben/flow.json (hunk 1 above)
Comment (ekl176) [reply app/flows/beratungshilfeFormular/finanzielleAngaben/flow.json:119]: good idea. I removed the guard completely and changed subsequent guards that relied on the partnerschaftOrSeperated to use hasPartnerschaftYes instead.

**Review by chohner — APPROVED** [review]
lgtm

**Pull request merged**